plugins/GSdx/GSRendererHW.cpp

```diff
@@ -207,15 +207,16 @@ int GSRendererHW::Interpolate_UV(float alpha, int t0, int t1)
 	return (int)t & ~0xF; // cheap rounding
 }
 
+//X1 not even used?? Should this be similar to alpha1 with different rounding?
 float GSRendererHW::alpha0(int L, int X0, int X1)
 {
-	float x = (X0 + 15) & ~0xF; // Round up
-	return (x - X0) / (float)L;
+	float x = float((X0 + 15) & ~0xF); // Round up
```

Comment:
Ok ill change that and remove the comment, just seemed strange we are passing X1 to it when we don't use it.